misc/docerina/src/main/java/org/ballerinalang/docgen/Generator.java

```diff
@@ -417,28 +391,48 @@ private static void addDocForObjectType(BLangObjectTypeNode objectType,
                 }
             }
         }
+        module.abstractObjects.add(new BAbstractObject(name, description, isDeprecated, fields, functions));
+    }
+
+    private static void addDocForClassType(BLangClassDefinition classDefinition, Module module) {
+        List<Function> functions = new ArrayList<>();
+        String name = classDefinition.getName().getValue();
+        String description = description(classDefinition);
+        boolean isDeprecated = isDeprecated(classDefinition.getAnnotationAttachments());
+
+        List<DefaultableVariable> fields = getFields(classDefinition, classDefinition.fields,
+                classDefinition.getMarkdownDocumentationAttachment(), module);
+
+        // Iterate through the functions
+        if (classDefinition.getFunctions().size() > 0) {
```

Comment:
@shehan360 do we need check? Below for each won't execute if there are 0 entries?

Reply:
Yes the check can be removed